.eslintrc.json

```diff
@@ -28,6 +29,24 @@
     "import/no-extraneous-dependencies": "error",
     "import/no-unassigned-import": "error",
     "import/no-unresolved": "off",
+    "import/order": ["error", { "newlines-between": "always-and-inside-groups" }],
+
+    "prettier/prettier": ["error", {
+      "semi": false,
+      "trailingComma": "all",
+      "printWidth": 90,
+      "quoteProps": "consistent",
+      "singleQuote": false,
+      "tabWidth": 2,
+      "useTabs": false,
+      "bracketSpacing": true,
+      "arrowParens": "always",
+      "proseWrap": "preserve",
+      "endOfLine": "lf"
+    }],
+
```

Comment:
@nicolasburtey why did you add prettier to eslint? 

We have prettier enforced already. Having it here adds a lot of noise while code is being typed.

Reply:
because I had issue with rebasing PR from my workflow. to make sure import are not duplicated or there are no import that are not used, I'm using `Shift + option + O` command from vs code. but this make the import order non really deterministic. 

to make it more deterministic and avoid a lot of rebase noise, I wanted to be able to use eslint as a formatter that can handle the import ordering. the best way I found to do it was to add prettier to eslint and then use eslint as the default formatter in VS Code. 

I tried other way to batch prettier + eslint from VS Code on save but didn't end up finding any workable solution

the new way is working a lot better for me. I'm saving files frequently to do the formatting basically. But it seems you have a different flow, and maybe you didn't had the issue I had previously